runtime/testy/stateManager.py

```diff
@@ -28,6 +28,9 @@ def initRobotState(self):
      "string1" : "abcde"
     }
 
+  def getRobotState(self):
+    self.processMapping[PROCESS_NAMES.STUDENT_CODE].send(self.state)
```

Comment:
oooh, this is dangerous. self.state may contain info we dont want students to access. 

Was there a reason to make this new function? why not just use the getValue function?